README.md

```diff
@@ -1,3 +1,91 @@
 # grisp_demo
 
 Demo application for GRiSP boards
+
+## Generate GRiSP EMMC Image
+
+**WARNING** This script has only been tested on Linux **WARNING**
```

Comment:
Idea: exit the script with an error if not on Linux.

Reply:
done